shared.primitives/README.md

```diff
@@ -0,0 +1,4 @@
+NaBL2 & Statix -- Java Components
+========================
+
+The project contains the Java primitives used by both Statix & NaBL2.
```

Comment:
Perhaps add that this code is for renaming.

Reply:
Done